src/esp/bindings/GfxBindings.cpp

```diff
@@ -61,6 +61,11 @@ void initGfxBindings(py::module& m) {
         Set this `Camera`'s projection matrix.
       )",
            "width"_a, "height"_a, "znear"_a, "zfar"_a, "hfov"_a)
+      .def("set_ortho_projection_matrix",
+           &RenderCamera::setOrthoProjectionMatrix, R"(
+        Set this `Orthographic Camera`'s projection matrix.
```

Comment:
Naming/consistency: why not have `set_orthographic_projection_matrix()` instead if there's already a`SensorSubType::ORTHOGRAPHIC`? Same for the viewer option, `--use-orthographic-camera`.

Or alternatively use the contraction everywhere. Having both short and long form seems like an unnecessary variation that hurts API usability.